settings.gradle.kts

```diff
@@ -1,3 +1,7 @@
 rootProject.name = "skiko-all"
```

Comment:
This file (the root `setting.gradle.kts` was added in the old times, when we had only the AWT implementation, and needed a way to open the project in IDEA as whole. Let's remove it to avoid confusion. Now we have a better way (opening individual examples)

But after removing, we should change the CI file (call gradlew not from the root folder, but from the `skiko` folder.